src/engine.rs

```diff
@@ -87,10 +87,18 @@ pub fn run(script_path: PathBuf, input_path: PathBuf) -> Result<FunctionRunResul
     let output: serde_json::Value = serde_json::from_slice(output.as_slice())
         .map_err(|e| anyhow!("Couldn't decode Script Output: {}", e))?;
 
+    // get the script file name
+    let name = script_path.file_name().unwrap().to_str().unwrap();
```

Comment:
The engine will throw an error way earlier in the code if the script is not a file etc. So I think its useless to add error handling here since it will never display the error.